presto-native-tests/src/test/java/com/facebook/presto/nativetests/TestRepartitionQueriesWithSmallPages.java

```diff
@@ -13,59 +13,44 @@
  */
 package com.facebook.presto.nativetests;
 
-import com.facebook.presto.nativeworker.NativeQueryRunnerUtils;
-import com.facebook.presto.nativeworker.PrestoNativeQueryRunnerUtils;
 import com.facebook.presto.testing.QueryRunner;
 import com.facebook.presto.tests.AbstractTestRepartitionQueries;
 import com.google.common.collect.ImmutableMap;
-import org.testng.annotations.Parameters;
+import org.testng.annotations.BeforeClass;
 
-import static com.facebook.presto.sidecar.NativeSidecarPluginQueryRunnerUtils.setupNativeSidecarPlugin;
+import java.util.Optional;
+
+import static com.facebook.presto.nativeworker.PrestoNativeQueryRunnerUtils.nativeHiveQueryRunnerBuilder;
 import static java.lang.Boolean.parseBoolean;
 
 public class TestRepartitionQueriesWithSmallPages
         extends AbstractTestRepartitionQueries
 {
-    @Parameters({"storageFormat", "sidecarEnabled"})
+    private String storageFormat;
+    private boolean sidecarEnabled;
+
+    @BeforeClass
+    @Override
+    public void init()
+            throws Exception
+    {
+        storageFormat = System.getProperty("storageFormat", "PARQUET");
+        sidecarEnabled = parseBoolean(System.getProperty("sidecarEnabled", "true"));
+        super.init();
+    }
+
     @Override
     protected QueryRunner createQueryRunner() throws Exception
     {
-        boolean sidecar = parseBoolean(System.getProperty("sidecarEnabled"));
-        QueryRunner queryRunner = PrestoNativeQueryRunnerUtils.nativeHiveQueryRunnerBuilder()
-                .setStorageFormat(System.getProperty("storageFormat"))
-                .setAddStorageFormatToPath(true)
-                .setUseThrift(true)
-                .setExtraProperties(
+        return NativeTestsUtils.createNativeQueryRunner(storageFormat, sidecarEnabled,
```

Comment:
I feel we needn't use the NativeTestsUtils method here... Passing the half built builder is quite awkward.

Reply:
Thanks for pointing it out, this was a bit hacky and reverted it as suggested.